javaslang/src/main/java/javaslang/collection/Traversable.java

```diff
@@ -631,7 +631,7 @@ default T last() {
         if (isEmpty()) {
             return Option.none();
         } else {
-            return ts.maxBy((o1, o2) -> ((Comparable<T>) o1).compareTo(o2));
+            return ts.maxBy(Comparators.naturalComparator());
```

Comment:
There was a case (in this class I think) where a type-hint was needed, i.e. `Comparators.<T> naturalComparator()`. Therefore I removed the import. Does not matter so much here...